### ISSUE TITLE
en-GB | 🇬🇧 | Word Addition/Removal Requests

### ISSUE THREAD

**Issue by SSoelvsten** [SSoelvsten/wordrow#38]
Please write in comments below, the words that you wish to be added or removed from the Danish dictionary. Alternatively, add a pull request, adding/removing that word or the rule that generates it.

Discussing Requests

To make this thread more manageable please refrain (if possible) to comment on other's request and instead just upvote or downvote them as follows.

    If you agree with a request, then add a 👍 to the comment in question.
    If you disagree and like to keep it, then add a 👎 to the comment in question.

**Comment by SSoelvsten** [comment]
Remove: **der**

**Comment by SSoelvsten** [comment]
Remove: All words with the *!* rule (NOSUGGEST). These are all foul words.

**Comment by SSoelvsten** [comment]
Remove: **rte**

**Comment by SSoelvsten** [comment]
Remove: **tos**

**Comment by SSoelvsten** [comment]
Remove: **soi**

**Comment by SSoelvsten** [comment]
Remove: **mah**

**Comment by SSoelvsten** [comment]
Remove: **ise**

**Comment by mlutze** [comment]
Remove:
* **esp**
* **upcase**

**Comment by mlutze** [comment]
Add:
* **eon**
* **eons**
* **lunes**

Remove:
* **les**
* **oles**

**Comment by mlutze** [comment]
Add
* rode
* laid

Remove
* est
* bas
* bes
* der
* odt
* rte
* del
* diam
* masc
* mascs
* cir
* pris
* amu
* mus

**Comment by mlutze** [comment]
Add
- targe

Remove
- spic*
- spick*
- spics*
- spicks*
- rte
- erat
- natl
- tif
- deg
- der
- wog*
- biog
- fas


\*Assuming we want to avoid offensive words

**Comment by SSoelvsten** [comment]
**Add**

- made

**Remove**

- atm
- mea
- matt

**Comment by SSoelvsten** [comment]
Remove:

- [ ] facie (always used together as "prima facie", so by itself I'm not sure it is a word)
- [ ] opp
- [ ] doc, docs
- [ ] deb (?)

**Comment by SSoelvsten** [comment]
Remove:

- [ ] woah
- [ ] phwoah
- [ ] pow
- [ ] pah

**Comment by SSoelvsten** [comment]
Add:
- [ ] sine

**Comment by mlutze** [comment]
Remove:
- [ ] ing
- [ ] nep
- [ ] nie
- [ ] ginn
- [ ] peni

**Comment by SSoelvsten** [comment]
From Collin's Dictionary:
```
ING	a meadow, esp. one beside a river [n -S]
NEP	(dialect) catmint, a plant attractive to cats [n -S]
NIE	(obsolete) to nigh, approach, also NIGH, NY, NYE [v NIED, NYING, NIES]
GINN	a class of spirits in Muslim folklore [n]
PENI	(Spenser) a penny, also PENIE [n -S]
ECOD	(archaic) a mild oath, also EGAD, EGADS, IGAD [interj]
```
So, I guess I should remove everything marked in the dictionary with *dialect*, *obsolete*, *spenser*, and *archaic*?

**Comment by mlutze** [comment]
Strange. None of those definitions came up when I followed the built-in link I'm the game.

**Comment by SSoelvsten** [comment]
Yeah, I feel kinda bummed out by Collin's Online Dictionary too. Some times they are better than Merriam Webster at the weird words they've added themselves, but then other times they are just completely useless.

**Comment by mlutze** [comment]
That would be very nice. I sometimes find myself with 5 extra tabs open, looking up several of the words.
